substanced/folder/static/css/mimetype-icons.css

```diff
@@ -0,0 +1,392 @@
+/*
```

Comment:
I don't get that:  why would having to keep two separate static registrations around be a good thing?  Not to mention the chance of having different versions slip in during maintenance.